pkg/cache/sql/informer/factory/informer_factory_test.go

```diff
@@ -63,7 +63,7 @@ func TestCacheFor(t *testing.T) {
 		fields := [][]string{{"something"}}
 		expectedGVK := schema.GroupVersionKind{}
 		sii := NewMockSharedIndexInformer(gomock.NewController(t))
-		sii.EXPECT().HasSynced().Return(true)
+		sii.EXPECT().HasSynced().Return(true).Times(2)
```

Comment:
Do we care that HasSynced has been called twice here? And are we sure that it will always be called twice? I just want to avoid flaky tests.

Reply:
ack, fixed in 4f36817